app/src/main/res/layout/activity_main.xml

```diff
@@ -8,12 +7,18 @@
     tools:context=".MainActivity">
 
 
-    <ListView
+    <!--<ListView
```

Comment:
Cool.
Also now onwards, do mention, which Issue a pull request refers to.
For example this PR is for #12